docs/using_lagoon/docker_images/nginx-drupal.md

```diff
@@ -0,0 +1,17 @@
+# nginx-drupal Image
+Lagoon `nginx-drupal` Docker image optimized to work with Drupal, based on Lagoon `nginx` image.
+
+## Lagoon & OpenShift adaptions
+This image is prepared to be used on Lagoon which leverages OpenShift.  
+There are therefore some things already done:
+
+- Folder permissions are automatically adapted with [`fix-permissions`](https://github.com/sclorg/s2i-base-container/blob/master/core/root/usr/bin/fix-permissions) so this image will work with a random user and therefore also on OpenShift.
+- For all file's section, there are two `include` directives: the `prepend*.conf` and the `append*.conf`. They allow to add and append custom configuration to `drupal.conf` file
```

Comment:
I think I would document this more clearer, the files are not called `prepend*.conf` but instead
- `/etc/nginx/conf.d/drupal/server_prepend*.conf;`
- `/etc/nginx/conf.d/drupal/location_prepend*.conf;`
- `/etc/nginx/conf.d/drupal/location_drupal_prepend*.conf;`

so I would go a bit into the detail what each of these files does.